src/Tests/Microsoft.DotNet.PackageValidation.Tests/ValidatePackageTargetTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Commands;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.PackageValidation.Tests
+{
+    public class ValidatePackageTargetTests : SdkTest
+    {
+        public ValidatePackageTargetTests(ITestOutputHelper log) : base(log)
+        {
+            // clearing the cache
+            string localSdkNugetCacheValidationPath = Path.Combine(TestContext.Current.NuGetCachePath, "microsoft.dotnet.packagevalidation");
+            Directory.Delete(localSdkNugetCacheValidationPath, recursive: true);
+
+            // Packing and copying the local version package validation
+            string packageValidationProjectPath = Path.Combine(Path.GetDirectoryName(_testAssetsManager.TestAssetsRoot), "Compatibility", "Microsoft.DotNet.PackageValidation", "Microsoft.DotNet.PackageValidation.csproj");
+            new PackCommand(Log, packageValidationProjectPath)
+                .Execute($"-p:PackageVersion=1.0.0-test;PackageOutputPath={TestContext.Current.TestPackages}");
+        }
+
+        [Fact]
+        public void ValidatePackageTargetRunsSuccessfully()
+        {
+            var testAsset = _testAssetsManager
+                .CopyTestAsset("PackageValidationTestProject")
+                .WithSource();
+
+            var result = new PackCommand(Log, Path.Combine(testAsset.TestRoot, "PackageValidationTestProject.csproj"))
+                .Execute();
+
+            // No failures while running the package validation on a simple assembly.
```

Comment:
is there a way to ensure package validation was indeed executed?

Reply:
Not in the current form. We can add some logging at then end of the task saying that the package validation run successfully.

Reply:
Are we adding this logging?

Reply:
Logging wont help here unless we print something to the output screen.

Reply:
Ah ok. So a low importance message can't be captured into the test outcome?

Reply:
no